### DISCUSSION TITLE
14 - Seed Registry

### DISCUSSION THREAD

**Discussion by kevinslin** [dendronhq/dendron#802] [RFCs/Ideas]
This is a discussion thread to talk about the [Dendron Seed Registry](https://wiki.dendron.so/notes/4039fc46-06b2-4f83-b817-fc490bafbcb3.html)

**Comment by brandoncroberts** [comment]
I had one question about the workspace topic. Have you checked out the npm workspaces feature that was released with npm V7? 
Seems like it could be a good looking term solution/ alternative to Lerna. 
Of course if you're totally happy with Lerna then probably not worth looking into changing

**Reply by brandoncroberts** [reply]
https://docs.npmjs.com/cli/v7/using-npm/workspaces#installing-workspaces

**Reply by kevinslin** [reply]
for workspaces, we're actually referring to Dendron [workspaces](https://wiki.dendron.so/notes/c4cf5519-f7c2-4a23-b93b-1c9a02880f6b.html) instead of lerna/yarn workspaces. dendron itself takes care of dependency management
